cluster/addons/kube-proxy/kube-proxy-ds.yaml

```diff
@@ -44,6 +44,22 @@ spec:
         - /bin/sh
         - -c
         - kube-proxy {{cluster_cidr}} --resource-container="" --oom-score-adj=-998 {{params}} 1>>/var/log/kube-proxy.log 2>&1
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10249
```

Comment:
10249 is for metric server. Should probably use 10256 (healthz port)?

Reply:
@MrHohn Works with 10249 too, but I will follow your suggestion.